src/components/cluster/Cluster.tsx

```diff
@@ -593,6 +595,50 @@ function ClusterForm({
         k8sversion: '',
     }
 
+    const ClusterInfoComponent = () => {
+        const k8sClusters = Object.values(CLUSTER_COMMAND)
+        return (
+            <>
+                {k8sClusters.map((cluster, key) => (
+                    <>
+                        <Tippy
+                            className="default-white p-0"
+                            theme="light"
+                            arrow={true}
+                            placement="bottom"
+                            trigger="click"
+                            interactive={true}
+                            content={
+                                <ClusterInfoStepsModal
+                                    subTitle={cluster.title}
+                                    command={cluster.command}
+                                    clusterName={cluster.clusterName}
+                                />
+                            }
+                            maxWidth="468px"
+                        >
+                            <span className="ml-4 mr-2 cb-5 cursor">{cluster.heading}</span>
+                        </Tippy>
+                        {key + 1 !== k8sClusters.length && <span className="cn-2">|</span>}
```

Comment:
can we write it like `{key < k8sClusters.length && <span className="cn-2">|</span>}`